rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "1.79"
+channel = "1.80"
```

Comment:
If I can believe my eyes, at least on windows only "1.80.0" works. Though that's from me hacking on probe-rs